docs/faq-debug.md

```diff
@@ -0,0 +1,16 @@
+# FAQ / Troubleshooting - Debugging/Development
+
+### Enabling Debug Logs
+
+How to enable more detailed debug logs in the extensions.
+If you need to report an issue attach these to give the most detailed information.
+
+1. Open the Command Palette (`cmd/ctrl` + `shift` + `p`), then search for "View Logs". Choose the correct option for the extension you want, eg: `AWS: View Logs` or `Amazon Q: View Logs`
```

Comment:
can we instead update https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#logging 

or implement a `Amazon Q: View Logs and Set Level to Debug` command, which would remove the need for most of these steps